doc/glm/meaning_difference.rst

```diff
@@ -0,0 +1,61 @@
+.. _meaning_difference:
+
+==================================================
+Difference in meanings between different toolboxes
+==================================================
+
+.. topic:: **Page summary**
+
+	* SPM has the same meaning of analysis levels, compared to nilearn's models
+		* First-level: analyze across runs
+		* Second-level: group-level analysis
+	* FSL has a different meaning
+		* First-level: analyze one session for one subject
+		* Second-level: analyze across sessions for one subject
+		* Third-level: group-level analysis
+	
+These differences can be put in the table below:
+
++----------------------------------------------+----------------+------------------+---------------+
+| Analyze                                      | nilearn        | SPM              | FSL           |
++==============================================+================+==================+===============+
+| One session, one subject                     | First-level           | First-level             | First\-level  |
++----------------------------------------------+----------------+------------------+---------------+
+| More than one session, one subject           | First\-level   | First\-level     | Second\-level |
++----------------------------------------------+----------------+------------------+---------------+
+| More than or equal to one session,           | Second\-level  | Second\-level    | Third\-level  |
+| more than one subject                        |                |                  |               |
++----------------------------------------------+----------------+------------------+---------------+
+
+*Table showing the differences of the meaning for level of models between toolboxes\/libraries*
+
+Statistical Parametric Mapping (SPM)
+====================================
+
+SPM uses the same notation as nilearn for analysis levels, with a note that a session still refers to an imaging session or a run, and within a run there could be multiple conditions (e.g., congruent and incongruent).
+In this case, `SPM`_ provided `tutorials`_ and documentation, including `lectures`_, which one could learn to analyze their own fMRI data with the meaning of analysis levels being as follows:
+
+	* `First-level analysis in SPM`_: Analyze across sessions for a subject (i.e., more than one session of one subject)
+	* `Second-level analysis in SPM`_: Analyze across several subjects (i.e., more than one subject with one or more sessions per subject). This is also known as **group-level analysis** which test if the average estimate across subjects is statistically significant.
+
+.. _SPM: https://www.fil.ion.ucl.ac.uk/spm/docs/
+.. _tutorials: https://www.fil.ion.ucl.ac.uk/spm/docs/tutorials/
+.. _lectures: https://www.fil.ion.ucl.ac.uk/spm/docs/courses/fmri_vbm/recordings/glm/
```

Comment:
Wonder if we should already start pointing to the new SPM documentation website:
see this VERY recent announcement https://github.com/spm/spm/discussions/25